js/main.js

```diff
@@ -40,5 +47,82 @@ function submitInfo(event) {
   $entryForm.reset();
 }
 
+// function to render entries
+function renderEntry(entry) {
+  // creates elements of DOM tree:
+  const $row = document.createElement('div');
+  $row.className = 'row entry-row';
+  const $column = document.createElement('div');
+  $column.className = 'column-full entry-column';
+  const $entryContainer = document.createElement('div');
+  $entryContainer.className = 'entry-container';
+  const $list = document.createElement('ul');
+  $list.className = 'journal-entry';
```

Comment:
You should not be creating any `ul` elements here. Your `ul` should exist in HTML, and you append the `li` to the `ul`